### PR TITLE
adding logs

### PR DESCRIPTION
**Pull request by TarasHryts**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -43,6 +59,43 @@
...
+                    <configLocation>${maven.checkstyle.plugin.configLocation}</configLocation>
+                    <encoding>UTF-8</encoding>
+                    <consoleOutput>true</consoleOutput>
+                    <failsOnError>true</failsOnError>
+                    <linkXRef>false</linkXRef>
+                </configuration>
+            </plugin>
+
+
+<!--            <plugin>-->
```
Comment (Sofasmile) [inline pom.xml]: remove commented code from repository

File: src/main/java/mate/academy/shop/anotation/AnnotatedClassMap.java [hunk 2]
```diff
@@ -4,19 +4,23 @@
...
 import mate.academy.shop.service.UserService;
-import mate.academy.shop.factory.Factory;
+import org.apache.log4j.Logger;
 
+import java.io.FileReader;
 import java.util.HashMap;
 import java.util.Map;
 
 public class AnnotatedClassMap {
+    final static Logger logger = Logger.getLogger(FileReader.class);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/shop/anotation/AnnotatedClassMap.java]: add access modifier
```suggestion
    final static Logger logger = Logger.getLogger(AnnotatedClassMap.class);
```

File: src/main/java/mate/academy/shop/anotation/Injector.java [hunk 3]
```diff
@@ -9,11 +12,13 @@
 import java.util.List;
 
 public class Injector {
+    final static Logger logger = Logger.getLogger(FileReader.class);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/shop/anotation/Injector.java]:
```suggestion
    private final static Logger logger = Logger.getLogger(Injector.class);
```
the same in with all loggers

File: src/main/java/mate/academy/shop/anotation/Injector.java [hunk 4]
```diff
@@ -9,11 +12,13 @@
...
     private static final String PROJECT_MAIN_PACKAGE = "mate.academy.shop";
     private static List<Class> classes = new ArrayList<>();
 
     static {
         try {
+            logger.info("Injector start working");
             classes.addAll(getClasses(PROJECT_MAIN_PACKAGE));
         } catch (ClassNotFoundException | IOException e) {
             e.printStackTrace();
```
Comment (Sofasmile) [inline src/main/java/mate/academy/shop/anotation/Injector.java]: remove e.printStackTrace(); and use logger

File: pom.xml (hunk 1 above)
Comment (TarasHryts) [reply pom.xml]: done

File: src/main/java/mate/academy/shop/anotation/AnnotatedClassMap.java (hunk 2 above)
Comment (TarasHryts) [reply src/main/java/mate/academy/shop/anotation/AnnotatedClassMap.java]: changed for all

File: src/main/java/mate/academy/shop/anotation/Injector.java (hunk 3 above)
Comment (TarasHryts) [reply src/main/java/mate/academy/shop/anotation/Injector.java]: changed for all

File: src/main/java/mate/academy/shop/anotation/Injector.java (hunk 4 above)
Comment (TarasHryts) [reply src/main/java/mate/academy/shop/anotation/Injector.java]: done

**Review by Sofasmile — APPROVED** [review]

**Pull request closed without merging**